webapp/client/App.js

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
 import PopularMovies from './PopularMovies';
 import Search from './Search'
+  ;
+
 class App extends React.Component {
-  constructor(props){
+  constructor(props) {
     super(props);
     this.state = {
       popular: {},
-      searchState: false
-    }
-    }
-    componentWillMount(){
-      fetch('/popular')
-      .then(results => {
-        return results.json()
-      }).then(data=> {
-        this.setState({popular: data})
-        //console.log(this.state.popular.results[0].title)
-      })
-    }
+      searchState: false,
+    };
+  }
+  componentWillMount() {
+    fetch('/popular')
+      .then(results => results.json()).then((data) => {
+        this.setState({ popular: data });
+        // console.log(this.state.popular.results[0].title)
+      });
```

Comment:
add .catch statement to handle errors. would be best if you could handle the error gracefully on the front end with some sort of display, telling the user that therehas been an error